xbout/tests/test_animate.py

```diff
@@ -32,7 +32,9 @@ class TestAnimate:
     def test_animate2D(self, create_test_file):
 
         save_dir, ds = create_test_file
-        animation = ds['n'].isel(y=1).bout.animate2D(y='z', save_as="%s/test" % save_dir)
+        animation = ds['n'].isel(x=1).bout.animate2D(save_as="%s/testyz" % save_dir)
+        animation = ds['n'].isel(y=2).bout.animate2D(save_as="%s/testxz" % save_dir)
+        animation = ds['n'].isel(z=3).bout.animate2D(save_as="%s/testxy" % save_dir)
```

Comment:
These aren't really testing different behaviour each time. All they have to do to pass is return some `animation` object.

We want to test that the dimensions are in the right order, so something like:
```python
assert animation.blocks[0].ax.xlabel == 'x'
```